docs/azure/TOC.yml

```diff
@@ -67,6 +67,8 @@
         href: ./sdk/authentication/azure-hosted-apps.md
       - name: Auth from on-premises apps
         href: ./sdk/authentication/on-premises-apps.md
+      - name: Auth via configuration files
+        href: ./sdk/authentication/create-token-credentials-from-configuration.md
```

Comment:
Do we show examples of [UseCredential](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.azure.azureclientfactorybuilder.usecredential?view=azure-dotnet) somewhere?

Reply:
It makes a couple appearances over at https://learn.microsoft.com/en-us/dotnet/azure/sdk/dependency-injection?tabs=web-app-builder#register-clients-and-subclients